src/main/java/com/playmonumenta/libraryofsouls/bestiary/BestiaryManager.java

```diff
@@ -43,8 +40,9 @@ public class BestiaryManager implements Listener {
 	 * Entity UUID key, Player UUID Set
 	 * The size here is automatically managed by removing the oldest entries as the map fills up
 	 */
-	private final Map<UUID, Set<UUID>> mDamageTracker = new LinkedHashMap<>(MAX_BOSS_TRACK_ENTRIES + 1, .75F, true) {
+	private final Map<UUID, Set<UUID>> mDamageTracker = new LinkedHashMap<UUID, Set<UUID>>(MAX_BOSS_TRACK_ENTRIES + 1, .75F, true) {
```

Comment:
IntelliJ actually complains if you specify the type when creating a new object if the type can be inferred from the variable declaration. I'd change this back to `new LinkedHashMap<>(...)`.

Reply:
It can't be inferred here - which is why I added it back. It won't compile without it, because this isn't creating a LinkedHashMap, it's creating a special object which extends linked hash map with custom on delete functionality